samples/config_single_check.yml

```diff
@@ -4,7 +4,7 @@ hosts:
     username : demo 
     passwd: demo123
 tests:
- - test_no_diff.yml 
+ - test_is_gt.yml 
```

Comment:
Please don't change the samples. Though harmless, this is not in line with the problem that this pull request aims to solve